composer/trainer/checkpoint.py

```diff
@@ -383,15 +384,14 @@ def should_checkpoint(self, state: State, event: Event) -> bool:
 
         return False
 
-    def save_checkpoint(self, state: State, seed: int, device: Device, config: Optional[Dict[str, Any]] = None) -> None:
+    def save_checkpoint(self, state: State, seed: int, device: Device) -> None:
         """Save the current state to a a new checkpoint file.
 
         Args:
             state (State): The current State of the trainer.
             seed (int): The seed used for random number generation.
             device (Device): The Device in use by this process.
             ddp (DDP): The DDP engine in use by this trainer.
```

Comment:
Tangential, but this looks obsolete.

Reply:
We really need a doc linter lol...thanks!